### PR TITLE
Add nix support

### PR DESCRIPTION
**Pull request by osa1**

### CODE REVIEW

**Comment by jschpp** [comment]
You need to add a test case for the extension.

**Comment by osa1** [comment]
@jschpp @ggreer anyone know what's wrong with my tests?

File: tests/list_file_types.t [hunk 1]
```diff
@@ -176,6 +176,9 @@ Language types are output:
   
     --nim
         .nim
+
```
Comment (k-takata) [inline tests/list_file_types.t]: Two spaces are needed in this line.

Comment (osa1) [inline tests/list_file_types.t]: What? Why? Other empty lines don't have two spaces.

**Comment by osa1** [comment]
Ready for landing @jschpp @ggreer

**Pull request merged**